debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java

```diff
@@ -258,7 +258,8 @@ protected String connectionString() {
 
     private Map<String, String> querySystemVariables(String statement) {
         Map<String, String> variables = new HashMap<>();
-        try (JdbcConnection mysql = jdbc.connect()) {
+        try {
+            JdbcConnection mysql = jdbc.connect();
             mysql.query(statement, rs -> {
```

Comment:
It seems the line before could just be removed and here it'd be `jdbc.query(...)`?

Reply:
done